server/views/pages/appointments/partials/appointment-journey-title.njk

```diff
@@ -0,0 +1,3 @@
+{% macro appointmentJourneyTitle(pageName, appointmentJourney) -%}
+  {{ "Appointments Management - " + appointmentJourney.mode | capitalize + " " + appointmentJourney.type | capitalize + " Appointment - " + pageName }}
```

Comment:
Not sure it's possible to test an njk macro with cheerio, or do you mean add a test to one of the pages it's implemented on?

Reply:
If it can't be tested as a macro, let's leave it. I think ideally there'd be a cheerio test for each view that pins these titles as they are very easily missed in manual QA. Can be done in another PR if agreed.